src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services10.php

```diff
@@ -53,11 +53,14 @@ public function getParameter(string $name): array|bool|string|int|float|\UnitEnu
         if (!(isset($this->parameters[$name]) || isset($this->loadedDynamicParameters[$name]) || \array_key_exists($name, $this->parameters))) {
             throw new ParameterNotFoundException($name);
         }
+
         if (isset($this->loadedDynamicParameters[$name])) {
-            return $this->loadedDynamicParameters[$name] ? $this->dynamicParameters[$name] : $this->getDynamicParameter($name);
+            $value = $this->loadedDynamicParameters[$name] ? $this->dynamicParameters[$name] : $this->getDynamicParameter($name);
+        } else {
+            $value = $this->parameters[$name];
         }
 
-        return $this->parameters[$name];
+        return $value;
```

Comment:
do all similar changes provide anything else that just CS changes? if yes I'd say we should revert

Reply:
it's not a CS change, but a refactoring to validate the `$value` when "non-empty parameters" are dumped, see diff in PhpDumper: https://github.com/symfony/symfony/pull/57611/files#diff-82d7ebdd6477cecb404b0d3f100a9026e950529edb9a665460ae0330dfad7080L1605-R1639

Note that the `if` block related to non-empty parameters is being removed here (that's why it looks like a CS change) as these test cases don't use the feature.

Reply:
See dumped file with non-empty parameters activated here: https://github.com/symfony/symfony/pull/57611/files#diff-4c69fc66a80dbb45b09a50078b9b97cc0bf1026ac5e293491f12f262964dbfbd